package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.5.25",
+  "version": "0.5.26",
```

Comment:
You also have to check in the package-lock.json file.

Reply:
Just this one last thing before this can be merged, please and thanks!

Reply:
done